roles/openshift_metrics/tasks/install_heapster.yaml

```diff
@@ -16,11 +16,9 @@
 - name: create heapster objects
   command: >
     {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ mktemp.stdout }}/templates/{{ file }}'
+    create -f '{{ mktemp.stdout }}/templates/{{ item }}'
```

Comment:
Can we create the object in one step at the end or is there a need to control cassandra before heapster?